apps/native-component-list/src/components/HeadingText.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { StyleSheet, Text, View, StyleProp, ViewStyle } from 'react-native';
+
+export default class HeadingText extends React.Component<{ style?: StyleProp<ViewStyle> }> {
```

Comment:
Shouldn't this somehow be handled by React already?

Reply:
Well, not really, `React.Component` is just plain `react`, while `StyleProp<ViewStyle>` is `react-native`. I haven't found any useful shortcut for providing styling the regular way. In `react-native.d.ts` every component that is exported has it's own `style?: StyleProps<ViewStyle>`.